lib/Reports/SingleUser.php

```diff
@@ -201,6 +203,22 @@ protected function getNumberOfSharesForUser($userId) {
 		return $numShares;
 	}
 
+	/**
+	 * @param string $userId
+	 * @return int
+	 */
+	protected function getUserDisplayName($userId) {
+		$query = $this->queries['displayName'];
```

Comment:
Instead of a query, I would propose to:
```
$user = $this->userManager->get($userId);
return $user->getDisplayName();
```

For this to work, you need to add `IUserManager $userManager` to the constructor of the class